datasources/media/metadata_test.go

```diff
@@ -118,7 +118,7 @@ func TestSplitCamelCaseIntoWords(t *testing.T) {
 		"GPSAltitudeRef":                   "GPS Altitude Ref",
 		"GPSAltitude":                      "GPS Altitude",
 		"GPSTimeStamp":                     "GPS Time Stamp",
-		"GPSSatelites":                     "GPS Satellites", // OK so it's misspelled in the EXIF data I encountered!
+		"GPSSatelites":                     "GPS Satelites", // OK so it's misspelled in the EXIF data I encountered!
```

Comment:
I actually want it spelled correctly in Timelinize though :upside_down_face:

Reply:
Ahhh I see 👍 , I'll revert that.

Reply:
@mholt I reverted the typo fix in https://github.com/timelinize/timelinize/pull/40/commits/4da830834f4db50858b4322db819efcfc19c4253, but that'll make the test `TestSplitCamelCaseIntoWords` fail. I've been trying to figure out a way to fix it, but that'd require also correct the spelling somehow in `splitCamelCaseIntoWords(s string)`, in addition to to the word splitting.

How do you see that working?

Reply:
On a second read, I'm not entirely sure I understand your intentions 🙃

Reply:
OH wow it's a test file. I didn't even notice that :laughing: I was just going through lint errors really fast.

We can just remove this line. That's totally my bad!

Reply:
Done! :wrench: